CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# dbt_fivetran_log v1.7.3
+[PR #126](https://github.com/fivetran/dbt_fivetran_log/pull/126) includes the following updates:
+
+## Performance Improvements
+- Updated the sequence of JSON parsing for model `fivetran_platform__audit_table` to reduce runtime. 
+
+## Bug Fixes
+- Updated model `fivetran_platform__audit_user_activity` to correct the JSON parsing used to determine column `email`.  
```

Comment:
could be good to note that was causing `fivetran_platform__audit_user_activity` to potentially have 0 rows